crypto/bls12381_utils.h

```diff
@@ -25,10 +25,12 @@ typedef uint8_t byte;
 #define Fp_BITS   381
 #define Fr_BITS   255
 #define Fp_BYTES  BITS_TO_BYTES(Fp_BITS)
-#define Fp_BYTES2 BITS_TO_BYTES(Fp_BITS)*2
+#define Fp2_BYTES (2*Fp_BYTES)
 #define Fp_DIGITS BITS_TO_DIGITS(Fp_BITS)
 #define Fr_BYTES  BITS_TO_BYTES(Fr_BITS)
 
+#define SEED_BYTES BITS_TO_BYTES(Fr_BITS + SEC_BITS)
```

Comment:
There are many seeds in the library and this length is particular for that `bn_randZr_star` usage. I think it would be better to keep it as a local seed length to that function.